src/selection.js

```diff
@@ -71,6 +71,16 @@ var Selection = (function() {
     },
 
     /**
+     * Check if a link can be added.
+     * If the editableHost or a parent is a link element linking
+     * should not be possible.
+     */
+    canLinkBeSet: function() {
+      var linkParents = content.getParentsAndSelf(this.host, 'a');
```

Comment:
@marcbachmann This is on purpose since nesting links should not be allowed in general. Especially in case of teasers the link tag can be outside of the editable.